control-plane-operator/controllers/hostedcontrolplane/v2/cno/component.go

```diff
@@ -110,6 +110,11 @@ type operand struct {
 }
 
 func checkOperandsRolloutStatus(cpContext component.WorkloadContext) (bool, error) {
+	// When NetworkType != OVNKubernetes, OVN-Kubernetes deployments won't exist, so we skip checking them.
```

Comment:
We do need to check if `multus-admission-controller` is deployed even for Calico.

Reply:
Updated.